.github/dependabot.yml

```diff
@@ -17,7 +19,22 @@ updates:
     commit-message:
       prefix: "[tune]"
       include: "scope"
-    # Only 3 upgrade PRs open at a time.
-    open-pull-requests-limit: 3
+    # Only 5 upgrade PRs open at a time.
+    open-pull-requests-limit: 5
     reviewers:
       - "ray-project/ray-tune"
+
+  # RLlib requirements
```

Comment:
@sven1977 you can modify this section however you see fit. You probably want to make a Github teams with other Rllib committers so its not always requesting your for review